teamwork/apps/projects/views/EditProjectView.py

```diff
@@ -115,7 +115,10 @@ def edit_project(request, slug):
         elif profAdded:
             messages.add_message(request, messages.SUCCESS, "Greppers have been added to the project.")
         else:
-            messages.add_message(request, messages.WARNING, "Failed to invite member(s) to project. Make sure they are enrolled in this course.")
+            if (not course in mem_courses):
+                messages.warning(request, "User failed to be added to the project. " + mem_to_add.username + " is not enrolled in the course")
+                # return False
+            #messages.add_message(request, messages.WARNING, "Student(s) is already added to the project.") 
```

Comment:
Put the message stating the student is already in the project in the else on this inner if